.travis.yml

```diff
@@ -13,10 +13,16 @@ matrix:
       script: ./tools/checkJSCurrent.sh
       node_js:
         - "10"
-    - name: "Old Release Check"
-      r: oldrel
-    - name: "Current Release Check"
```

Comment:
Is it worth keeping this one around in case we forget to manually update? Maybe we could have this instead of 3.6 for now and then when we upgrade to 3.7 we'd just add 3.6 to our list of old instances to test on?

Reply:
Sure, that sounds reasonable.